nautobot/dcim/models/devices.py

```diff
@@ -393,6 +406,15 @@ class Platform(OrganizationalModel):
         null=True,
         help_text="Optionally limit this platform to devices of a certain manufacturer",
     )
+    network_driver = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text=mark_safe(
+            'The <a href="https://netutils.readthedocs.io/en/latest/user/lib_use_cases_lib_mapper/">network driver</a> '
+            "to use when interacting with devices, e.g. <code>cisco_ios</code>, <code>arista_eos</code>, etc.<br>"
+            "Library-specific driver names will be derived from this setting as appropriate"
```

Comment:
I think we need to be specific that this is expected to be the `netutils` _normalized_ name, as I read this implementation. Maybe even call this `netutils_normalized_network_driver`.

Reply:
> I think we need to be specific that this is expected to be the netutils normalized name, as I read this implementation. 

Yes and agreed. 

> Maybe even call this netutils_normalized_network_driver.

My vote would be to keep as is